cni/pkg/cmd/root.go

```diff
@@ -99,11 +99,12 @@ var rootCmd = &cobra.Command{
 			log.Infof("Starting ambient node agent with inpod redirect mode on socket %s", cniEventAddr)
 			ambientAgent, err := nodeagent.NewServer(ctx, watchServerReady, cniEventAddr,
 				nodeagent.AmbientArgs{
```

Comment:
I've found that disable+enable ambient has errors with this pr:

```
[pod/istio-cni-node-xflzt/install-cni] 2024-11-18T19:36:10.478130Z      info    cni-agent       Namespace default is disabled from ambient mesh
[pod/istio-cni-node-xflzt/install-cni] 2024-11-18T19:36:10.478533Z      info    Running command (without lock by env and nss): iptables-legacy -t mangle -D PREROUTING -j ISTIO_PRERT
[pod/istio-cni-node-xflzt/install-cni] 2024-11-18T19:36:10.480383Z      info    Running command (without lock by env and nss): iptables-legacy -t mangle -D OUTPUT -j ISTIO_OUTPUT
[pod/istio-cni-node-xflzt/install-cni] 2024-11-18T19:36:10.481933Z      info    Running command (without lock by env and nss): iptables-legacy -t nat -D OUTPUT -j ISTIO_OUTPUT
[pod/istio-cni-node-xflzt/install-cni] 2024-11-18T19:36:10.483118Z      info    Running command (without lock by env and nss): iptables-legacy -t raw -D PREROUTING -j ISTIO_PRERT
[pod/istio-cni-node-xflzt/install-cni] 2024-11-18T19:36:10.483930Z      error   Command error output: xtables parameter problem: Couldn't load target `ISTIO_PRERT':No such file or directory
[pod/istio-cni-node-xflzt/install-cni]
[pod/istio-cni-node-xflzt/install-cni] Try `iptables -h' or 'iptables --help' for more information.
[pod/istio-cni-node-xflzt/install-cni] 2024-11-18T19:36:10.483954Z      info    Running command (without lock by env and nss): iptables-legacy -t raw -D OUTPUT -j ISTIO_OUTPUT
[pod/istio-cni-node-xflzt/install-cni] 2024-11-18T19:36:10.484736Z      error   Command error output: xtables parameter problem: Couldn't load target `ISTIO_OUTPUT':No such file or directory
[pod/istio-cni-node-xflzt/install-cni]
[pod/istio-cni-node-xflzt/install-cni] Try `iptables -h' or 'iptables --help' for more information.
[pod/istio-cni-node-xflzt/install-cni] 2024-11-18T19:36:10.484765Z      info    Running command (without lock by env and nss): iptables-legacy -t nat -D PREROUTING -j ISTIO_PRERT
[pod/istio-cni-node-xflzt/install-cni] 2024-11-18T19:36:10.485810Z      info    Running command (without lock by env and nss): iptables-legacy -t mangle -F ISTIO_PRERT
[pod/istio-cni-node-xflzt/install-cni] 2024-11-18T19:36:10.486612Z      info    Running command (without lock by env and nss): iptables-legacy -t mangle -F ISTIO_OUTPUT
[pod/istio-cni-node-xflzt/install-cni] 2024-11-18T19:36:10.487435Z      info    Running command (without lock by env and nss): iptables-legacy -t nat -F ISTIO_PRERT
[pod/istio-cni-node-xflzt/install-cni] 2024-11-18T19:36:10.488348Z      info    Running command (without lock by env and nss): iptables-legacy -t nat -F ISTIO_OUTPUT
[pod/istio-cni-node-xflzt/install-cni] 2024-11-18T19:36:10.489175Z      info    Running command (without lock by env and nss): iptables-legacy -t raw -F ISTIO_PRERT
[pod/istio-cni-node-xflzt/install-cni] 2024-11-18T19:36:10.490024Z      error   Command error output: xtables other problem: No chain/target/match by that name.
[pod/istio-cni-node-xflzt/install-cni] 2024-11-18T19:36:10.490052Z      info    Running command (without lock by env and nss): iptables-legacy -t raw -F ISTIO_OUTPUT
[pod/istio-cni-node-xflzt/install-cni] 2024-11-18T19:36:10.490766Z      error   Command error output: xtables other problem: No chain/target/match by that name.
[pod/istio-cni-node-xflzt/install-cni] 2024-11-18T19:36:10.490794Z      info    Running command (without lock by env and nss): iptables-legacy -t mangle -X ISTIO_PRERT
[pod/istio-cni-node-xflzt/install-cni] 2024-11-18T19:36:10.491864Z      info    Running command (without lock by env and nss): iptables-legacy -t mangle -X ISTIO_OUTPUT
[pod/istio-cni-node-xflzt/install-cni] 2024-11-18T19:36:10.492976Z      info    Running command (without lock by env and nss): iptables-legacy -t nat -X ISTIO_PRERT
[pod/istio-cni-node-xflzt/install-cni] 2024-11-18T19:36:10.493824Z      info    Running command (without lock by env and nss): iptables-legacy -t nat -X ISTIO_OUTPUT
[pod/istio-cni-node-xflzt/install-cni] 2024-11-18T19:36:10.494622Z      info    Running command (without lock by env and nss): iptables-legacy -t raw -X ISTIO_PRERT
[pod/istio-cni-node-xflzt/install-cni] 2024-11-18T19:36:10.495405Z      error   Command error output: xtables other problem: No chain/target/match by that name.
[pod/istio-cni-node-xflzt/install-cni] 2024-11-18T19:36:10.495448Z      info    Running command (without lock by env and nss): iptables-legacy -t raw -X ISTIO_OUTPUT
[pod/istio-cni-node-xflzt/install-cni] 2024-11-18T19:36:10.496460Z      error   Command error output: xtables other problem: No chain/target/match by that name.
[pod/istio-cni-node-xflzt/install-cni] 2024-11-18T19:36:10.496502Z      info    Running command (without lock by env and nss): ip6tables-legacy -t mangle -D PREROUTING -j ISTIO_PRERT
[pod/istio-cni-node-xflzt/install-cni] 2024-11-18T19:36:10.498174Z      info    Running command (without lock by env and nss): ip6tables-legacy -t mangle -D OUTPUT -j ISTIO_OUTPUT
[pod/istio-cni-node-xflzt/install-cni] 2024-11-18T19:36:10.499158Z      info    Running command (without lock by env and nss): ip6tables-legacy -t nat -D OUTPUT -j ISTIO_OUTPUT
[pod/istio-cni-node-xflzt/install-cni] 2024-11-18T19:36:10.500103Z      info    Running command (without lock by env and nss): ip6tables-legacy -t raw -D PREROUTING -j ISTIO_PRERT
[pod/istio-cni-node-xflzt/install-cni] 2024-11-18T19:36:10.501506Z      error   Command error output: xtables parameter problem: Couldn't load target `ISTIO_PRERT':No such file or directory
[pod/istio-cni-node-xflzt/install-cni]
[pod/istio-cni-node-xflzt/install-cni] Try `ip6tables -h' or 'ip6tables --help' for more information.
[pod/istio-cni-node-xflzt/install-cni] 2024-11-18T19:36:10.501553Z      info    Running command (without lock by env and nss): ip6tables-legacy -t raw -D OUTPUT -j ISTIO_OUTPUT
[pod/istio-cni-node-xflzt/install-cni] 2024-11-18T19:36:10.502628Z      error   Command error output: xtables parameter problem: Couldn't load target `ISTIO_OUTPUT':No such file or directory
[pod/istio-cni-node-xflzt/install-cni]
[pod/istio-cni-node-xflzt/install-cni] Try `ip6tables -h' or 'ip6tables --help' for more information.
[pod/istio-cni-node-xflzt/install-cni] 2024-11-18T19:36:10.502681Z      info    Running command (without lock by env and nss): ip6tables-legacy -t nat -D PREROUTING -j ISTIO_PRERT
[pod/istio-cni-node-xflzt/install-cni] 2024-11-18T19:36:10.503895Z      info    Running command (without lock by env and nss): ip6tables-legacy -t mangle -F ISTIO_PRERT
[pod/istio-cni-node-xflzt/install-cni] 2024-11-18T19:36:10.504856Z      info    Running command (without lock by env and nss): ip6tables-legacy -t mangle -F ISTIO_OUTPUT
[pod/istio-cni-node-xflzt/install-cni] 2024-11-18T19:36:10.505948Z      info    Running command (without lock by env and nss): ip6tables-legacy -t nat -F ISTIO_PRERT
[pod/istio-cni-node-xflzt/install-cni] 2024-11-18T19:36:10.506966Z      info    Running command (without lock by env and nss): ip6tables-legacy -t nat -F ISTIO_OUTPUT
[pod/istio-cni-node-xflzt/install-cni] 2024-11-18T19:36:10.508052Z      info    Running command (without lock by env and nss): ip6tables-legacy -t raw -F ISTIO_PRERT
[pod/istio-cni-node-xflzt/install-cni] 2024-11-18T19:36:10.508918Z      error   Command error output: xtables other problem: No chain/target/match by that name.
[pod/istio-cni-node-xflzt/install-cni] 2024-11-18T19:36:10.508948Z      info    Running command (without lock by env and nss): ip6tables-legacy -t raw -F ISTIO_OUTPUT
[pod/istio-cni-node-xflzt/install-cni] 2024-11-18T19:36:10.509699Z      error   Command error output: xtables other problem: No chain/target/match by that name.
[pod/istio-cni-node-xflzt/install-cni] 2024-11-18T19:36:10.509729Z      info    Running command (without lock by env and nss): ip6tables-legacy -t mangle -X ISTIO_PRERT
[pod/istio-cni-node-xflzt/install-cni] 2024-11-18T19:36:10.511074Z      info    Running command (without lock by env and nss): ip6tables-legacy -t mangle -X ISTIO_OUTPUT
[pod/istio-cni-node-xflzt/install-cni] 2024-11-18T19:36:10.512096Z      info    Running command (without lock by env and nss): ip6tables-legacy -t nat -X ISTIO_PRERT
[pod/istio-cni-node-xflzt/install-cni] 2024-11-18T19:36:10.513296Z      info    Running command (without lock by env and nss): ip6tables-legacy -t nat -X ISTIO_OUTPUT
[pod/istio-cni-node-xflzt/install-cni] 2024-11-18T19:36:10.514322Z      info    Running command (without lock by env and nss): ip6tables-legacy -t raw -X ISTIO_PRERT
[pod/istio-cni-node-xflzt/install-cni] 2024-11-18T19:36:10.515125Z      error   Command error output: xtables other problem: No chain/target/match by that name.
[pod/istio-cni-node-xflzt/install-cni] 2024-11-18T19:36:10.515155Z      info    Running command (without lock by env and nss): ip6tables-legacy -t raw -X ISTIO_OUTPUT
[pod/istio-cni-node-xflzt/install-cni] 2024-11-18T19:36:10.516425Z      error   Command error output: xtables other problem: No chain/target/match by that name.
[pod/istio-cni-node-xflzt/install-cni] 2024-11-18T19:36:10.523102Z      info    Running command (without lock by env and nss): iptables-legacy -t mangle -D PREROUTING -j ISTIO_PRERT
[pod/istio-cni-node-xflzt/install-cni] 2024-11-18T19:36:10.524404Z      info    Running command (without lock by env and nss): iptables-legacy -t mangle -D OUTPUT -j ISTIO_OUTPUT
[pod/istio-cni-node-xflzt/install-cni] 2024-11-18T19:36:10.525468Z      info    Running command (without lock by env and nss): iptables-legacy -t nat -D OUTPUT -j ISTIO_OUTPUT
[pod/istio-cni-node-xflzt/install-cni] 2024-11-18T19:36:10.526484Z      info    Running command (without lock by env and nss): iptables-legacy -t raw -D PREROUTING -j ISTIO_PRERT
[pod/istio-cni-node-xflzt/install-cni] 2024-11-18T19:36:10.527232Z      error   Command error output: xtables parameter problem: Couldn't load target `ISTIO_PRERT':No such file or directory
[pod/istio-cni-node-xflzt/install-cni]
[pod/istio-cni-node-xflzt/install-cni] Try `iptables -h' or 'iptables --help' for more information.
[pod/istio-cni-node-xflzt/install-cni] 2024-11-18T19:36:10.527254Z      info    Running command (without lock by env and nss): iptables-legacy -t raw -D OUTPUT -j ISTIO_OUTPUT
[pod/istio-cni-node-xflzt/install-cni] 2024-11-18T19:36:10.528028Z      error   Command error output: xtables parameter problem: Couldn't load target `ISTIO_OUTPUT':No such file or directory
[pod/istio-cni-node-xflzt/install-cni]
[pod/istio-cni-node-xflzt/install-cni] Try `iptables -h' or 'iptables --help' for more information.
[pod/istio-cni-node-xflzt/install-cni] 2024-11-18T19:36:10.528056Z      info    Running command (without lock by env and nss): iptables-legacy -t nat -D PREROUTING -j ISTIO_PRERT
[pod/istio-cni-node-xflzt/install-cni] 2024-11-18T19:36:10.529278Z      info    Running command (without lock by env and nss): iptables-legacy -t mangle -F ISTIO_PRERT
[pod/istio-cni-node-xflzt/install-cni] 2024-11-18T19:36:10.530267Z      info    Running command (without lock by env and nss): iptables-legacy -t mangle -F ISTIO_OUTPUT
[pod/istio-cni-node-xflzt/install-cni] 2024-11-18T19:36:10.531560Z      info    Running command (without lock by env and nss): iptables-legacy -t nat -F ISTIO_PRERT
[pod/istio-cni-node-xflzt/install-cni] 2024-11-18T19:36:10.532952Z      info    Running command (without lock by env and nss): iptables-legacy -t nat -F ISTIO_OUTPUT
[pod/istio-cni-node-xflzt/install-cni] 2024-11-18T19:36:10.533769Z      info    Running command (without lock by env and nss): iptables-legacy -t raw -F ISTIO_PRERT
[pod/istio-cni-node-xflzt/install-cni] 2024-11-18T19:36:10.534547Z      error   Command error output: xtables other problem: No chain/target/match by that name.
[pod/istio-cni-node-xflzt/install-cni] 2024-11-18T19:36:10.534569Z      info    Running command (without lock by env and nss): iptables-legacy -t raw -F ISTIO_OUTPUT
[pod/istio-cni-node-xflzt/install-cni] 2024-11-18T19:36:10.535360Z      error   Command error output: xtables other problem: No chain/target/match by that name.
[pod/istio-cni-node-xflzt/install-cni] 2024-11-18T19:36:10.535386Z      info    Running command (without lock by env and nss): iptables-legacy -t mangle -X ISTIO_PRERT
[pod/istio-cni-node-xflzt/install-cni] 2024-11-18T19:36:10.536177Z      info    Running command (without lock by env and nss): iptables-legacy -t mangle -X ISTIO_OUTPUT
[pod/istio-cni-node-xflzt/install-cni] 2024-11-18T19:36:10.536958Z      info    Running command (without lock by env and nss): iptables-legacy -t nat -X ISTIO_PRERT
[pod/istio-cni-node-xflzt/install-cni] 2024-11-18T19:36:10.537729Z      info    Running command (without lock by env and nss): iptables-legacy -t nat -X ISTIO_OUTPUT
[pod/istio-cni-node-xflzt/install-cni] 2024-11-18T19:36:10.538518Z      info    Running command (without lock by env and nss): iptables-legacy -t raw -X ISTIO_PRERT
[pod/istio-cni-node-xflzt/install-cni] 2024-11-18T19:36:10.539149Z      error   Command error output: xtables other problem: No chain/target/match by that name.
[pod/istio-cni-node-xflzt/install-cni] 2024-11-18T19:36:10.539177Z      info    Running command (without lock by env and nss): iptables-legacy -t raw -X ISTIO_OUTPUT
[pod/istio-cni-node-xflzt/install-cni] 2024-11-18T19:36:10.539795Z      error   Command error output: xtables other problem: No chain/target/match by that name.
[pod/istio-cni-node-xflzt/install-cni] 2024-11-18T19:36:10.539821Z      info    Running command (without lock by env and nss): ip6tables-legacy -t mangle -D PREROUTING -j ISTIO_PRERT
[pod/istio-cni-node-xflzt/install-cni] 2024-11-18T19:36:10.540672Z      info    Running command (without lock by env and nss): ip6tables-legacy -t mangle -D OUTPUT -j ISTIO_OUTPUT
[pod/istio-cni-node-xflzt/install-cni] 2024-11-18T19:36:10.541380Z      info    Running command (without lock by env and nss): ip6tables-legacy -t nat -D OUTPUT -j ISTIO_OUTPUT
[pod/istio-cni-node-xflzt/install-cni] 2024-11-18T19:36:10.542164Z      info    Running command (without lock by env and nss): ip6tables-legacy -t raw -D PREROUTING -j ISTIO_PRERT
[pod/istio-cni-node-xflzt/install-cni] 2024-11-18T19:36:10.542807Z      error   Command error output: xtables parameter problem: Couldn't load target `ISTIO_PRERT':No such file or directory
[pod/istio-cni-node-xflzt/install-cni]
[pod/istio-cni-node-xflzt/install-cni] Try `ip6tables -h' or 'ip6tables --help' for more information.
[pod/istio-cni-node-xflzt/install-cni] 2024-11-18T19:36:10.542827Z      info    Running command (without lock by env and nss): ip6tables-legacy -t raw -D OUTPUT -j ISTIO_OUTPUT
[pod/istio-cni-node-xflzt/install-cni] 2024-11-18T19:36:10.543660Z      error   Command error output: xtables parameter problem: Couldn't load target `ISTIO_OUTPUT':No such file or directory
[pod/istio-cni-node-xflzt/install-cni]
[pod/istio-cni-node-xflzt/install-cni] Try `ip6tables -h' or 'ip6tables --help' for more information.
[pod/istio-cni-node-xflzt/install-cni] 2024-11-18T19:36:10.543686Z      info    Running command (without lock by env and nss): ip6tables-legacy -t nat -D PREROUTING -j ISTIO_PRERT
[pod/istio-cni-node-xflzt/install-cni] 2024-11-18T19:36:10.545027Z      info    Running command (without lock by env and nss): ip6tables-legacy -t mangle -F ISTIO_PRERT
[pod/istio-cni-node-xflzt/install-cni] 2024-11-18T19:36:10.546136Z      info    Running command (without lock by env and nss): ip6tables-legacy -t mangle -F ISTIO_OUTPUT
[pod/istio-cni-node-xflzt/install-cni] 2024-11-18T19:36:10.547099Z      info    Running command (without lock by env and nss): ip6tables-legacy -t nat -F ISTIO_PRERT
[pod/istio-cni-node-xflzt/install-cni] 2024-11-18T19:36:10.548435Z      info    Running command (without lock by env and nss): ip6tables-legacy -t nat -F ISTIO_OUTPUT
[pod/istio-cni-node-xflzt/install-cni] 2024-11-18T19:36:10.549563Z      info    Running command (without lock by env and nss): ip6tables-legacy -t raw -F ISTIO_PRERT
[pod/istio-cni-node-xflzt/install-cni] 2024-11-18T19:36:10.550225Z      error   Command error output: xtables other problem: No chain/target/match by that name.
[pod/istio-cni-node-xflzt/install-cni] 2024-11-18T19:36:10.550245Z      info    Running command (without lock by env and nss): ip6tables-legacy -t raw -F ISTIO_OUTPUT
[pod/istio-cni-node-xflzt/install-cni] 2024-11-18T19:36:10.550892Z      error   Command error output: xtables other problem: No chain/target/match by that name.
[pod/istio-cni-node-xflzt/install-cni] 2024-11-18T19:36:10.550906Z      info    Running command (without lock by env and nss): ip6tables-legacy -t mangle -X ISTIO_PRERT
[pod/istio-cni-node-xflzt/install-cni] 2024-11-18T19:36:10.551558Z      info    Running command (without lock by env and nss): ip6tables-legacy -t mangle -X ISTIO_OUTPUT
[pod/istio-cni-node-xflzt/install-cni] 2024-11-18T19:36:10.552206Z      info    Running command (without lock by env and nss): ip6tables-legacy -t nat -X ISTIO_PRERT
[pod/istio-cni-node-xflzt/install-cni] 2024-11-18T19:36:10.552869Z      info    Running command (without lock by env and nss): ip6tables-legacy -t nat -X ISTIO_OUTPUT
[pod/istio-cni-node-xflzt/install-cni] 2024-11-18T19:36:10.553616Z      info    Running command (without lock by env and nss): ip6tables-legacy -t raw -X ISTIO_PRERT
[pod/istio-cni-node-xflzt/install-cni] 2024-11-18T19:36:10.554309Z      error   Command error output: xtables other problem: No chain/target/match by that name.
[pod/istio-cni-node-xflzt/install-cni] 2024-11-18T19:36:10.554323Z      info    Running command (without lock by env and nss): ip6tables-legacy -t raw -X ISTIO_OUTPUT
[pod/istio-cni-node-xflzt/install-cni] 2024-11-18T19:36:10.554984Z      error   Command error output: xtables other problem: No chain/target/match by that name.
```

Reply:
I'm not sure if this is triggered by my PR, as it involves the DeleteInpodRules() function, which I believe behaves this way by default.
Depending on the iptables configuration (such as tproxy or ingressMode), some chains or rules are not created. Therefore, I think the delete commands in DeleteInpodRules() are causing these log errors, which are ultimately ignored.

I will reproduce and take a deeper look

Reply:
This should now be fixed after the rebase, as every delete command is optional and executed using `ext.RunQuietlyAndIgnore()`.

Reply:
```
2024-11-22T19:07:17.818679Z     info    cni-agent       Namespace default is disabled from ambient mesh
2024-11-22T19:07:17.819072Z     info    Running command (without nss): iptables-nft -t mangle -D PREROUTING -j ISTIO_PRERT
2024-11-22T19:07:17.846139Z     info    Running command (without nss): iptables-nft -t mangle -D OUTPUT -j ISTIO_OUTPUT
2024-11-22T19:07:17.899352Z     info    Running command (without nss): iptables-nft -t nat -D OUTPUT -j ISTIO_OUTPUT
2024-11-22T19:07:17.929391Z     info    Running command (without nss): iptables-nft -t raw -D PREROUTING -j ISTIO_PRERT
2024-11-22T19:07:17.930416Z     info    Running command (without nss): iptables-nft -t raw -D OUTPUT -j ISTIO_OUTPUT
2024-11-22T19:07:17.931394Z     info    Running command (without nss): iptables-nft -t nat -D PREROUTING -j ISTIO_PRERT
2024-11-22T19:07:17.959558Z     info    Running command (without nss): iptables-nft -t mangle -F ISTIO_PRERT
2024-11-22T19:07:17.989366Z     info    Running command (without nss): iptables-nft -t mangle -F ISTIO_OUTPUT
2024-11-22T19:07:18.022704Z     info    Running command (without nss): iptables-nft -t nat -F ISTIO_PRERT
2024-11-22T19:07:18.049434Z     info    Running command (without nss): iptables-nft -t nat -F ISTIO_OUTPUT
2024-11-22T19:07:18.089415Z     info    Running command (without nss): iptables-nft -t raw -F ISTIO_PRERT
2024-11-22T19:07:18.090516Z     info    Running command (without nss): iptables-nft -t raw -F ISTIO_OUTPUT
2024-11-22T19:07:18.091485Z     info    Running command (without nss): iptables-nft -t mangle -X ISTIO_PRERT
2024-11-22T19:07:18.116129Z     info    Running command (without nss): iptables-nft -t mangle -X ISTIO_OUTPUT
2024-11-22T19:07:18.149391Z     info    Running command (without nss): iptables-nft -t nat -X ISTIO_PRERT
2024-11-22T19:07:18.169395Z     info    Running command (without nss): iptables-nft -t nat -X ISTIO_OUTPUT
2024-11-22T19:07:18.219360Z     info    Running command (without nss): iptables-nft -t raw -X ISTIO_PRERT
2024-11-22T19:07:18.220441Z     info    Running command (without nss): iptables-nft -t raw -X ISTIO_OUTPUT
2024-11-22T19:07:18.221456Z     info    Running command (without nss): ip6tables-nft -t mangle -D PREROUTING -j ISTIO_PRERT
2024-11-22T19:07:18.242739Z     info    Running command (without nss): ip6tables-nft -t mangle -D OUTPUT -j ISTIO_OUTPUT
2024-11-22T19:07:18.272692Z     info    Running command (without nss): ip6tables-nft -t nat -D OUTPUT -j ISTIO_OUTPUT
2024-11-22T19:07:18.309346Z     info    Running command (without nss): ip6tables-nft -t raw -D PREROUTING -j ISTIO_PRERT
2024-11-22T19:07:18.310372Z     info    Running command (without nss): ip6tables-nft -t raw -D OUTPUT -j ISTIO_OUTPUT
2024-11-22T19:07:18.311274Z     info    Running command (without nss): ip6tables-nft -t nat -D PREROUTING -j ISTIO_PRERT
2024-11-22T19:07:18.332695Z     info    Running command (without nss): ip6tables-nft -t mangle -F ISTIO_PRERT
2024-11-22T19:07:18.357760Z     info    Running command (without nss): ip6tables-nft -t mangle -F ISTIO_OUTPUT
2024-11-22T19:07:18.386101Z     info    Running command (without nss): ip6tables-nft -t nat -F ISTIO_PRERT
2024-11-22T19:07:18.409373Z     info    Running command (without nss): ip6tables-nft -t nat -F ISTIO_OUTPUT
2024-11-22T19:07:18.429485Z     info    Running command (without nss): ip6tables-nft -t raw -F ISTIO_PRERT
2024-11-22T19:07:18.430658Z     info    Running command (without nss): ip6tables-nft -t raw -F ISTIO_OUTPUT
2024-11-22T19:07:18.431716Z     info    Running command (without nss): ip6tables-nft -t mangle -X ISTIO_PRERT
2024-11-22T19:07:18.462712Z     info    Running command (without nss): ip6tables-nft -t mangle -X ISTIO_OUTPUT
2024-11-22T19:07:18.496074Z     info    Running command (without nss): ip6tables-nft -t nat -X ISTIO_PRERT
2024-11-22T19:07:18.519346Z     info    Running command (without nss): ip6tables-nft -t nat -X ISTIO_OUTPUT
2024-11-22T19:07:18.546076Z     info    Running command (without nss): ip6tables-nft -t raw -X ISTIO_PRERT
2024-11-22T19:07:18.547162Z     info    Running command (without nss): ip6tables-nft -t raw -X ISTIO_OUTPUT
```

Reply:
(namely, fix was: https://github.com/istio/istio/pull/54015 which was merged separately for 1.24.1)